src/server/lib/graphql.js

```diff
@@ -0,0 +1,154 @@
+import { GraphQLClient } from 'graphql-request';
```

Comment:
This file should not be commit. If you're looking for adding the `extraInfo` field on the invoice, I suggest to do it in https://github.com/opencollective/opencollective-invoices/blob/d53b048a97727c3f014ca671eab55fe1cfaa7b09/lib/graphql/fragments.js#L3.